domains/certificates/Query.API/API/MeasurementsSyncer/Metrics/MeasurementSyncMetrics.cs

```diff
@@ -19,6 +19,16 @@ public class MeasurementSyncMetrics : IMeasurementSyncMetrics
     private long recoveredMeasurements = 0;
     private ObservableCounter<long> RecoveredMeasurements { get; }
 
```

Comment:
Kan vi få en total measurements metric også?

Reply:
There already is ett_certificate_measurements_fetched